src/SFA.DAS.LevyTransferMatching.Web/Models/Opportunities/DetailViewModel.cs

```diff
@@ -0,0 +1,11 @@
+﻿using SFA.DAS.LevyTransferMatching.Web.Models.Shared;
+
+namespace SFA.DAS.LevyTransferMatching.Web.Models.Opportunities
+{
+    public class DetailViewModel : OpportunitiesPostRequest
+    {
+        public Opportunity Opportunity { get; set; }
```

Comment:
The type was used in a previous view to list opportunities (see `OpportunitiesController.Index`). Given the model provided the same level of information required for this view, I reused it, rather than copying and pasting property names.

I _can_ copy and paste the properties into the view model directly, if you like? Please let me know.

Reply:
Based on your latter comments, I'll remove this and copy and paste the properties instead.

Reply:
Properties duplicated/structure flattened.